### ISSUE TITLE
No way to delete Sandbox instance

### ISSUE THREAD

**Issue by deboer-tim** [redhat-developer/podman-desktop-sandbox-ext#66]
If I successfully create a Sandbox instance, the Delete button is disabled. To delete I need to wait for the status not to be running (e.g. wait until tomorrow?).

It should be possible to stop/delete the instance whenever I am done using it.

**Comment by dgolovin** [comment]
@deboer-tim I agree, but the provider API controls lifecycle commands (delete is lifecycle command) based on connection status. Delete is enabled only when connections is stopped or unknown.

**Comment by deboer-tim** [comment]
Ok, so maybe we need an API update to distinguish between local instances (that can be started/stopped and deleted) vs remote ones where you can just drop the connection (delete the context)?
